Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -933,10 +933,7 @@ export class BotProject {
 
   private isReferred = (LUFile: LuFile) => {
     const dialogs = this.dialogs;
-    if (dialogs.findIndex(dialog => dialog.luFile === LUFile.id) !== -1) {
-      return true;
-    }
-    return false;
+    return !!~dialogs.findIndex(dialog => dialog.luFile === Path.basename(LUFile.id, `.${this.locale}`));
```

Comment:
So [alllocale].*.lu would be publish at this moment?

Reply:
update: publish all locale files now